exla/lib/exla.ex

```diff
@@ -294,12 +294,13 @@ defmodule EXLA do
 
   ## Examples
 
+      iex> fun = fn a, b -> Nx.add(a, b) end
```

Comment:
Shouldn't this be the same as &Nx.add/2?

Reply:
Ok, so from what I gathered the BEAM will treat captures differently than `fn` functions, is that right?

Reply:
Yeah, an optimization in the future may break it, but they should be distinct.